Dockerfile

```diff
@@ -37,8 +41,9 @@ ENV HELM_PLUGINS /opt/helm-plugins
 
 COPY --from=builder /workspace/manager .
 COPY --from=builder /workspace/helm-plugins ${HELM_PLUGINS}
-
-COPY build/charts/ /charts/
+COPY --from=builder /workspace/charts .
+COPY --from=builder /workspace/config .
+COPY --from=builder /workspace/manifests .
```

Comment:
These would not be needed, as there is no deployment happening from within the container.

Reply:
Removed